docs/resources/alerting_channel.md

```diff
@@ -0,0 +1,53 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "squaredup_alerting_channel Resource - terraform-provider-squaredup"
+subcategory: ""
+description: |-
+  SquaredUp Alerting Channel
+---
+
+# squaredup_alerting_channel (Resource)
+
+SquaredUp Alerting Channel
+
+## Example Usage
+
+```terraform
+data "squaredup_alerting_channel_types" "example" {
+  display_name = "Slack API"
+}
+
+resource "squaredup_alerting_channel" "slack_api_alert" {
```

Comment:
Probably worth including the optional `description` property for this resource as the API already supports it even if the UI doesn't yet expose it.

Reply:
I have added this on https://github.com/squaredup/terraform-provider-squaredup/pull/25/commits/1038d77dd7e4265ccceeb33d4169e2a59a6979d6